CHANGELOG_API.md

```diff
@@ -0,0 +1,54 @@
+## API v0 alpha
```

Comment:
?

```suggestion
## API v1 alpha
```

And further on below it says beta:

```suggestion
## API v1 beta
```

Reply:
I choose to name the current API `v0`, and because it was an alpha version I added `alpha` to the name. The new API is labelled `v1`. I think I will drop the `alpha` and `beta` mention because it is confusing.

The order in the changelog may be confusion too (oldest first), will change it.

Reply:
Done.